sherpa/astro/tests/test_astro_data_notebook.py

```diff
@@ -47,7 +57,11 @@ def check(r, summary, name, label, nmeta):
     """Very limited checks of the structure"""
     assert r is not None
 
-    if plot.backend.name == 'pylab':
+    # If not HAS_PYLAB, then the symbol PylabBackend is not defined
+    # and test collection will fail. So, we short-circuit that
+    # such that PylabBackend will only  be needed it
```

Comment:
```suggestion
    # such that PylabBackend will only be needed if
```

Reply:
Thanks for the suggestion. I'm closing this here, because I'm doing more rewording of the sentence and will fix that at the same time.